src/OpenLoco/Ui/WindowManager.h

```diff
@@ -7,6 +7,7 @@
 #include "../Viewport.hpp"
 #include "../Window.h"
 #include <cstddef>
+#include <functional>
```

Comment:
Why was this header introduced?

Reply:
This was needed for std::function which was previously got through one of the other headers.